geomstats/geometry/spd_matrices_space.py

```diff
@@ -717,3 +717,145 @@ def exp_domain(self, tangent_vec, base_point):
         domain = gs.concatenate((inf_value, sup_value), axis=1)
 
         return domain
+
+
+class SPDMetricLogEuclidean(RiemannianMetric):
+
+    def __init__(self, n):
+        dimension = int(n * (n + 1) / 2)
+        super(SPDMetricLogEuclidean, self).__init__(
+            dimension=dimension,
+            signature=(dimension, 0, 0))
+        self.n = n
+        self.space = SPDMatricesSpace(n)
+
+    def inner_product(self, tangent_vec_a, tangent_vec_b, base_point):
+        """
```

Comment:
Fix docstring format:
"""Compute the inner...

(See template shared by @cshewmake2 , this allows to generate the documentation website automatically).